core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -783,21 +662,33 @@ sealed trait Chunk[+A] { self =>
 }
 
 object Chunk {
+  //For priority on specialized methods
+  implicit class ChunkOps[A](private val self: Chunk[A]) extends AnyVal {
```

Comment:
I think this can go away per comment above.

Reply:
I wish it would go away as well! 😃(maybe losing the features is fine)

Reply:
ChunkOps is removed.